migrator/src/main/java/com/google/phonenumbers/migrator/MigrationFactory.java

```diff
@@ -56,21 +56,24 @@ public static MigrationJob createMigration(String number, String country) throws
         .orElseThrow(() -> new RuntimeException(
             "Country code " + countryCode+ " not supported in metadata"));
 
-    return new MigrationJob(numberRanges, countryCode, recipes, ranges);
+    return new MigrationJob(numberRanges, countryCode, recipes, ranges, /*lenientExport=*/false);
```

Comment:
You can add a space between the first star, the equals and the last slash, ie. 

/* lenientExport= */ false

Reply:
Here and below